routes/account.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express");
+const router = express.Router();
+const Account = require("../models/Accounts");
+router.post('/add', (req, res) => {
+  try {
+    const account = new Account({ title: req.body.title, description: req.body.description });
+    account.save();
+
+    res.status(200).json(account);
+  } catch (err) {
+    res.status(400).json(err.message);
+  }
+})
+
+
+router.get('/', async (req, res) => {
+  const account = await Account.find({});
```

Comment:
this logic should be in controller. route should look like `router.get('/', Account.find)`